src/Components/ArticleCard.js

```diff
@@ -1,20 +1,20 @@
 import "./ArticleCard.css";
 
-const ArticleCard = (props) => {
-  const date = props.articles.created_at.substring(0, 10);
+const ArticleCard = ({articles, onClick}) => {
+  const date = articles.created_at.substring(0, 10);
 
   return (
-    <div className="articlecard-main__container">
+    <div key={articles.article_id} className="articlecard-main__container" onClick={onClick}>
       <div className="articlecard-image__container">
-        <img src={props.articles.article_img_url} alt="article image" />
+        <img src={articles.article_img_url} alt="article image" />
       </div>
       <div className="articlecard-body__container">
         <div className="articlecard-date-author__container">
-          <div>{props.articles.author}</div>
+          <div>{articles.author}</div>
           <div>{date}</div>
         </div>
```

Comment:
Similarly to a previous comment, don't use divs for text - consider whether it is a heading, paragraph, inline text for example